Modules/xASL_module_ASL.m

```diff
@@ -755,18 +755,47 @@
     fprintf('\n%s', ['-> Detected ' xASL_num2str(nLists) ' list(s) for concatenating sessions']);
 end
 
+% Read and check a corresponding list with merging scalings
+if ~isfield(x.modules.asl, 'SessionMergingScaling')
+	% By default set as empty
+	x.modules.asl.SessionMergingScaling = {};
+else
+	% The parameter should be a list of lists, so like for SessionMerginList - we need to convert it if needed
+	if ~isempty(x.modules.asl.SessionMergingScaling) && ~iscell(x.modules.asl.SessionMergingScaling)
+		x.modules.asl.SessionMergingScaling = {x.modules.asl.SessionMergingScaling};
+	end
+end
+
+if ~isempty(x.modules.asl.SessionMergingList) && isempty(x.modules.asl.SessionMergingScaling)
+	% If the merging list is not empty, but scaling list is, then fill with all 1s in a field of the same length
+	for iMergingList = 1:length(x.modules.asl.SessionMergingList)
+		x.modules.asl.SessionMergingScaling{iMergingList} = ones(1, length(x.modules.asl.SessionMergingList{iMergingList}));
+	end
+end
+
+% Compare the two lists and report an error if the lengths do not correspond
```

Comment:
Note how much easier it is to have a FA vector instead of this quick and dirty manual parameter that can go wrong on many levels ;)

Reply:
We haven't figured out how to make it work with FA alone. Otherwise, this would have of course been the number 1 solution.